From bbe55896a92a20db2ce79d100e3027cd4940ddce Mon Sep 17 00:00:00 2001
From: igo95862 <igo95862@yandex.ru>
Date: Mon, 11 Feb 2019 14:53:23 -0800
Subject: [PATCH] Fixed iscsi.service considering every signal and exit code as
 successful. Now only code 21 (no objects found to execute on) and normal exit
 conditions are valid.

---
 etc/systemd/iscsi.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/systemd/iscsi.service b/etc/systemd/iscsi.service
index 03876c5..e475888 100644
--- a/etc/systemd/iscsi.service
+++ b/etc/systemd/iscsi.service
@@ -8,7 +8,7 @@ ConditionPathExists=/etc/iscsi/initiatorname.iscsi
 
 [Service]
 Type=oneshot
-ExecStart=-/sbin/iscsiadm -m node --loginall=automatic
+ExecStart=/sbin/iscsiadm -m node --loginall=automatic
 ExecStop=/sbin/iscsiadm -m node --logoutall=automatic
 ExecStop=/sbin/iscsiadm -m node --logoutall=manual
 SuccessExitStatus=21
-- 
1.8.3.1

